jobs/config.json

```diff
@@ -4020,12 +4020,14 @@
       "--check-version-skew=false",
       "--env-file=jobs/platform/gce.env",
       "--env-file=jobs/env/ci-kubernetes-e2e-gci-gce-latest-downgrade-kube-proxy-ds.env",
-      "--extract=ci/latest-1.7",
       "--extract=ci/latest",
+      "--extract=ci/k8s-stable1",
       "--gcp-zone=us-central1-f",
       "--provider=gce",
-      "--test_args=--ginkgo.focus=\\[Feature:KubeProxyDaemonSetDowngrade\\] --upgrade-target=ci/latest-1.7 --upgrade-image=gci --minStartupPods=8",
-      "--timeout=90m"
+      "--skew",
+      "--test_args=--ginkgo.focus=\\[Feature:KubeProxyDaemonSetDowngrade\\] --upgrade-target=ci/k8s-stable1 --upgrade-image=gci --minStartupPods=8",
+      "--timeout=120m",
```

Comment:
bump the timeout in prow/config.yaml as well

Reply:
Thanks! It would be great if the test error could say it clearly...

Reply:
:joy: :joy: :joy: yup you figured.. I'll try to locate this error message..